_includes/pbc/all/install-features/202400.0/install-the-warehouse-user-management-feature.md

```diff
@@ -245,14 +257,277 @@ Ensure that the plugins work correctly:
+     */
+    protected function getScopeProviderPlugins(): array
+    {
+        return [
+            new WarehouseOauthScopeProviderPlugin(),
+        ];
+    }
+
+    /**
+     * @return list<\Spryker\Zed\OauthExtension\Dependency\Plugin\OauthRequestGrantTypeConfigurationProviderPluginInterface>
+     */
+    protected function getOauthRequestGrantTypeConfigurationProviderPlugins(): array
+    {
+        return [
+            new WarehouseOauthRequestGrantTypeConfigurationProviderPlugin(),
+        ];
+    }
+}
+```
+</details>
+
+{% info_block warningBox "Verification" %}
+
+1. In the Back Office, go to **Users&nbsp;<span aria-label="and then">></span> Users**. This opens **USERS LIST**.
+2. For a user of your choice, in **Actions**, click **Edit**.
+3. On the **Edit User: `{USER_NAME}`** page that opens, select **THIS USER IS A WAREHOUSE USER**. The `{USER_NAME}` placeholder stands for the name of the user selected in the previous step.
+4. Click **Update**.
+5. Go back to **USERS LIST**.
+6. For the user edited in step 2, in **Actions**, click **Assign Warehouses**. This opens the **Assign Warehouse to User: `{USER_NAME}`** page. The `{USER_NAME}` placeholder stands for the name of the user you're assigning a warehouse to.
+7. In the **Select warehouses to assign** tab, for a needed warehouse, select **ASSIGN** and click **Save**.
+8. Authenticate the user while requesting a token:
+
+   ```json
+   POST /access-tokens HTTP/1.1
+   Host: glue-backend.mysprykershop.com
+   Content-Type: application/vnd.api+json
+   Content-Length: 167
+   {
+      "data": {
+         "type": "access-tokens",
+         "attributes": {
+            "username": "{USERNAME}",
+            "password": "{PASSWORD}"
+         },
+         "links": {
+            "self": "https://glue-backend.mysprykershop.com/access-tokens"
+         }
+      }
+   }
+   ```
+
+9. Generate a warehouse token with the generated token from the previous step:
+
+   ```json
+   POST /warehouse-tokens HTTP/1.1
+   Host: glue-backend.mysprykershop.com
+   Content-Type: application/vnd.api+json
+   Content-Length: 165
+   {
+      "data": {
+         "type": "warehouse-tokens",
+         "links": {
+            "self": "https://glue-backend.mysprykershop.com/warehouse-tokens"
+         }
+      }
+   }
+   ```
+
+{% endinfo_block %}
+
+3. Enable the Backend API resource by registering the plugin:
+
+| PLUGIN                                 | SPECIFICATION                                         | PREREQUISITES | NAMESPACE                                                              |
+|----------------------------------------|-------------------------------------------------------|---------------|------------------------------------------------------------------------|
+| WarehouseUserAssignmentsResourcePlugin | Registers the `warehouse-user-assignments` resource.  |               | Spryker\Glue\WarehouseUsersBackendApi\Plugin\GlueBackendApiApplication |
```

Comment:
* We have change to the new name "WarehouseUserAssignmentsBackendResourcePlugin".